eng/scripts/test.ps1

```diff
@@ -0,0 +1,74 @@
+#Requires -Version 7.0
```

Comment:
The name of this file made me think this was some kind of temporary test script.  Can we call it run-tests or something like this so it sounds more purposeful?

Reply:
I renamed the scripts to:
- Initialize-Repository.ps1
- Build-Packages.ps1
- Test-Packages.ps1